src/Common/Core/Impl/Install/RInstallation.cs

```diff
@@ -123,33 +144,96 @@ public static class RInstallation {
             return path;
         }
 
-        public static void GoToRInstallPage() {
-            Process.Start("https://mran.revolutionanalytics.com/download");
+        public static Exception LaunchRClientSetup() {
```

Comment:
If it's solely to display the error, and all callers do that, then perhaps this should be done right here? Then there's nothing to return.